azure-pipelines.yml

```diff
@@ -284,4 +285,70 @@ jobs:
       IntegrationTestsStorageAccountName: $(IntegrationTestsStorageAccountName)
       IntegrationTestsStorageAccountKey: $(IntegrationTestsStorageAccountKey)
     displayName: 'Upload Core Tools build to the storage account for Integration Tests'
-    condition: eq(variables.IsIntegrationBuild, 'true')
\ No newline at end of file
+    condition: eq(variables.IsIntegrationBuild, 'true')
+
+- job: LinuxPackage
+  condition: ne(variables['LinuxPackageBuildTag'], '')
+  timeoutInMinutes: "90"
+  pool:
```

Comment:
i think it should be

pool:
    name: '1ES-Hosted-AzFunc'
    demands:
      - ImageOverride -equals MMS2022TLS

Reply:
refer line 19/20

Reply:
We will address it as part of other ticket.